Source/WebCore/rendering/RenderLayer.h

```diff
@@ -593,6 +593,10 @@ class RenderLayer final : public CanMakeSingleThreadWeakPtr<RenderLayer>, public
     bool isVisuallyNonEmpty(PaintedContentRequest* = nullptr) const;
     // True if this layer container renderers that paint.
     void determineNonLayerDescendantsPaintedContent(PaintedContentRequest&) const;
+#if HAVE(HDR_SUPPORT)
+    // True of if renderer itself draws HDR content, no traversal is done.
+    bool drawsHDRContent() const;
```

Comment:
Please use consistent terminology with `paintsHDRContent` / `paintHDRContentOnly`. I think this is the "only" version? A good name would make the comment unnecessary. This is just for replaced elements (image, canvas etc), so you could call it `isReplacedElementWithHDR` or something.

Reply:
This function was renamed `isReplacedElementWithHDR()`.